### PR TITLE
[MOSIP-29035] Updated helm with role and rolebindings to automate post deployment onboarding steps

### PR DESCRIPTION
**Pull request by akilalakshmanan**

### CODE REVIEW

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — CHANGES_REQUESTED** [review]
namespace to be passed from helm values

**Review by syedsalman3753 — APPROVED** [review]

**Review by syedsalman3753 — APPROVED** [review]

File: charts/partner-onboarder/templates/role.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .Release.Name }}-deployment-role
```
Comment (ckm007) [inline charts/partner-onboarder/templates/role.yaml:4]: {{ .Release.Name }}-deployment-role --> {{ .Release.Name }}-role

File: charts/partner-onboarder/values.yaml [hunk 2]
```diff
@@ -443,10 +443,15 @@ onboarding:
...
       enabled: true
+    - name: mimoto-oidc
+      enabled: false
 
   configmaps:
     s3:
       s3-host: 'http://minio.minio:9000'
       s3-user-key: 'admin'
       s3-region: ''
+    onboarder-namespace:
```
Comment (ckm007) [inline charts/partner-onboarder/values.yaml:454]: onboarder-namespace?

Comment (akilalakshmanan) [reply charts/partner-onboarder/values.yaml:454]: As per Syed's suggestion to pass the namespace values to the default.sh of dockerfile through helm chart, I have added these changes

**Review by ckm007 — APPROVED** [review]

**Pull request merged**